server/@types/licenceApiImport/index.d.ts

```diff
@@ -88,41 +88,25 @@ export interface paths {
 
 export interface components {
   schemas: {
-    /** @description Request object for updating the COM responsible for an offender */
+    /** Request object for updating the COM responsible for an offender */
     UpdateComRequest: {
-      /**
-       * Format: int64
-       * @description The unique identifier of the COM, retrieved from Delius
-       * @example 22003829
-       */
+      /** The unique identifier of the COM, retrieved from Delius */
       staffIdentifier: number
-      /**
-       * @description The Delius username for the COM
-       * @example jbloggs
-       */
+      /** The Delius username for the COM */
       staffUsername: string
-      /**
-       * @description The email address of the COM
```

Comment:
We must have different configurations or something? Mine generates these `description` and `example` tags